### ISSUE TITLE
[glyf table] Clarify meaning of scale01 and scale10 fields

### ISSUE THREAD

**Issue by Lorp** [MicrosoftDocs/typography-issues#1129]
Due to confusion about labelling of transform matrix elements in various graphics systems it would be good to improve clarity on the _scale01_ and _scale10_ fields in the [glyf table](https://learn.microsoft.com/en-us/typography/opentype/spec/glyf).

As I wrote [here](https://github.com/googlefonts/fontc/pull/721#issuecomment-1944155086), it would help enormously to document the matrix itself using the local terms.

So in the glyf spec we would add:

```math
\begin{bmatrix}
    xscale & scale10 \\
    scale01 & yscale
\end{bmatrix}
```

Thus for a 90º CCW rotation…

```math
\begin{bmatrix}
    0 & -1 \\
    1 & 0
\end{bmatrix}
```

… we have:

```
xscale = 0
scale01 = 1
scale10 = -1
yscale = 0
```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b2a9a6a1-a9b9-0864-8982-322da821389e
* Version Independent ID: a42612e3-dc1d-f980-fe4a-4880d1fd0086
* Content: [glyf - Glyf data table (OpenType 1.9) - Typography](https://learn.microsoft.com/en-us/typography/opentype/spec/glyf)
* Content Source: [typographydocs/opentype/spec/glyf.md](https://github.com/MicrosoftDocs/typography/blob/live/typographydocs/opentype/spec/glyf.md)
* Product: **typography**
* GitHub Login: @PeterCon
* Microsoft Alias: **alib**

**Comment by anthrotype** [comment]
the Apple TrueType reference has more to say about about how to interpret the component transform but admittedly it is not much clearer (scroll down to 'Compound glyphs' section of https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6glyf.html)

**Comment by PeterCon** [comment]
Perhaps a description could be used similar to that used for Affine2x3 (used in PaintTransform) in COLR?

**Comment by PeterCon** [comment]
This is fixed in OT 1.9.1. See the [OT 1.9.1 beta](https://docs.microsoft.com/en-us/typography/opentype/otspec191/ot191beta) for draft revisions addressing this issue. Closing.
